### ISSUE TITLE
Cannot build nimble on Windows?

### ISSUE THREAD

**Issue by anta40** [nim-lang/nimble#104]
Hi,

I tried to build nimble on Windows, and it failed:
CC: nimble_packageinfo
CC: stdlib_parsecfg
CC: nimble_version
CC: nimble_tools
CC: nimble_nimbletypes
CC: nimble_download
CC: nimble_config
[Linking]
C:/mingw/bin/../lib/gcc/i686-w64-mingw32/4.9.2/../../../../i686-w64-mingw32/bin/ld.exe: cannot open output file c:\users
\csl-nb-064\downloads\software\nimrod\nimble-git\src\nimble.exe: Permission denied
collect2.exe: error: ld returned 1 exit status
Error: execution of an external program failed
Traceback (most recent call last)
nimble.nim(871)          nimble
nimble.nim(845)          doAction
nimble.nim(604)          install
nimble.nim(496)          installFromDir
nimble.nim(445)          buildFromDir
tools.nim(20)            doCmd
Error: unhandled exception: Execution failed with exit code 1 [NimbleError]
Error: execution of an external program failed

Some info:
1. I put all of the Nimrod stuffs in C:\Users\CSL-NB-064\Downloads\Software\Nimrod.
2. My Nimrod compiler's git directory C:\Users\CSL-NB-064\Downloads\Software\Nimrod\nim-git
3. Nimble's git directory is C:\Users\CSL-NB-064\Downloads\Software\Nimrod\nimble-git
4. I always succed in building Nimrod
5. This is the GCC i'm using: http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.9.2/threads-posix/sjlj/i686-4.9.2-release-posix-sjlj-rt_v3-rev1.7z/download

**Comment by dom96** [comment]
Have you read this? https://github.com/nim-lang/nimble#from-source

TL;DR: You need to rename your nimble exe for this to work.
